### DISCUSSION TITLE
Synchronizing Multiple Audio Sources?

### DISCUSSION THREAD

**Discussion by gantttt** [mozilla/hubs#3226] [General Questions & Discussion] [answered]
Hi all! 

Apologies, not sure if this should live in the Spoke or Hubs section, but:

I'm working on building a virtual sound installation in Hubs, but currently having some trouble synchronizing different audio instantiations from Spoke -- I've created a 'virtual 8 channel' speaker set up with 8 individual audio sources in the scene from Spoke, but I've noticed that when I enter the scene, they tend to auto-play/instantiate at slightly different times, desynchronizing the audio. 

Is there any clever way I could fix this/work around it? many thanks!

matthew

**Comment by pattersonbl2** [comment, marked as answer]
Unfortunately, this is very difficult to configure because media sources load/play at different times. I would suggest using one audio source throughout the scene. Synchronizing one media source across clients is already difficult.

**Reply by sobralfreenat** [reply]
I am also keep trying to solve the same issue.

**Reply by sobralfreenat** [reply]
Maybe something as group and play group w or w/o delay?
